package.json

```diff
@@ -20,21 +20,23 @@
   "license": "MIT",
   "dependencies": {
     "alexa-app": "^3.0.0",
-    "alexa-verifier-middleware": "^0.1.8",
+    "alexa-verifier-middleware": "^0.1.9",
```

Comment:
I don't think this is a required change, the library works fine with 0.1.8, doesn't it?